lagom-persistence-couchbase/core/src/main/resources/play/reference-overrides.conf

```
@@ -5,3 +5,9 @@ akka.actor.default-mailbox.stash-capacity = 10000
 akka.persistence.journal.plugin = couchbase-journal.write
 akka.persistence.snapshot-store.plugin = couchbase-journal.snapshot
 
+
+# bucket must be provided by application.conf
+couchbase-journal.write.bucket = null
```

does this really work, I'd have guessed it just sets the bucket name to the string "null"?

---

How is this done for the Lagom Cassandra? Isn't Lagom defining a default keyspace/table?

---

@renatocaval is this how you guys do it?

---

Or @ignasi35

---

https://github.com/lagom/lagom/pull/663/files#diff-c6b25c0104c43e1d17e89d36d6acc502R33

We had a default and fell back to setting the default value in `reference-conf` to `null` to fail fast when users were not setting that value.

---

> does this really work, I'd have guessed it just sets the bucket name to the string "null"?

Yes, it works like this:
`Configuration key 'write.bucket' is set to null but expected STRING`

I'll add a validation similar to one in Lagom that @ignasi35 has referenced.